### PR TITLE
Sudhanshu/cart

### PR DESCRIPTION
**Pull request by SudhanshuGulhane**
updated the hardcoded urls, also on home page displaying the categories and products is implemented

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -33,7 +33,8 @@ export default {
 
       //fetch categories
       await axios.get(this.baseURL + "category/")
-      .then(res => this.categories = res.data)
+      .then(res => {this.categories = res.data
+                    console.log(res.data)})
```
Comment (shivamag00) [inline src/App.vue]: Remove console log

File: src/components/Category/CategoryCard.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="card category-card">
-    <img :src="category.imageUrl" class="card-img-top" alt="image" @click="listProducts">
+    <img :src="this.imageUrl" class="card-img-top" alt="image" @click="listProducts">
```
Comment (shivamag00) [inline src/components/Category/CategoryCard.vue]: Why are you using this instead of category?

File: src/components/Product/ProductBox.vue [hunk 3]
```diff
@@ -2,7 +2,7 @@
   <div class="product-box">
       <div class="row">
         <div class="col-4" @click="showDetails">
-            <img class="img-fluid" v-bind:src="product.imageURL" alt="product-image">
+            <img class="img-fluid" v-bind:src="this.imgUrl" alt="product-image">
```
Comment (shivamag00) [inline src/components/Product/ProductBox.vue]: why are using this?

File: src/components/Product/ProductBox.vue [hunk 4]
```diff
@@ -19,11 +19,19 @@
 <script>
 export default {
     name : "ProductBox",
-    props : ["product"],
+    props : ["baseURL","product"],
```
Comment (shivamag00) [inline src/components/Product/ProductBox.vue]: Are you using baseURL in this component?

File: src/components/Product/ProductCard.vue [hunk 5]
```diff
@@ -11,7 +11,7 @@
 <script>
 export default {
   name : "ProductCard",
-  props : ["product"],
+  props : ["baseURL","product"],
```
Comment (shivamag00) [inline src/components/Product/ProductCard.vue]: Aye you using baseURL here?

File: src/components/Category/CategoryCard.vue (hunk 2 above)
Comment (SudhanshuGulhane) [reply src/components/Category/CategoryCard.vue]: I am binding the data ... 
data(){
    return{
      imageUrl:''
    }
  },
  mounted(){
    this.imageUrl = this.category.imageUrl
  }

File: src/components/Product/ProductBox.vue (hunk 3 above)
Comment (SudhanshuGulhane) [reply src/components/Product/ProductBox.vue]: here too same
data(){
        return{
            imgUrl:''
        }
    },
    mounted(){
        this.imgUrl = this.product.imageURL
    }

File: src/components/Product/ProductBox.vue (hunk 4 above)
Comment (SudhanshuGulhane) [reply src/components/Product/ProductBox.vue]: yes we dont need it ,i'll remove it

File: src/components/Product/ProductCard.vue (hunk 5 above)
Comment (SudhanshuGulhane) [reply src/components/Product/ProductCard.vue]: yes we dont need it ,i'll remove it

**Pull request closed without merging**